### PR TITLE
update label text on sort menu for all collections page

### PR DESCRIPTION
**Pull request by avertrees**
## Ticket:

- JIRA ticket [include "sort by" label inside the drop down](https://newyorkpubliclibrary.atlassian.net/browse/DR-3323)

## This PR does the following:

-

## Open Questions

-

## Accessibility concerns or updates

-

### Checklist:

- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.
- [ ] I have updated the CHANGELOG.md.

### CODE REVIEW

File: app/src/components/pages/collectionsPage/collectionsPage.tsx [hunk 1]
```diff
@@ -159,11 +160,12 @@ export function CollectionsPage({ data, params, renderCollections }) {
             onChange: (e) => handleSearchChange(e),
           }}
           onSubmit={handleSearchSubmit}
-          labelText={""}
+          labelText="Search Collections by title"
+          aria-label="Search Collections by title"
```
Comment (7emansell) [inline app/src/components/pages/collectionsPage/collectionsPage.tsx:164]: major nit but should collections be capitalized

**Review by 7emansell — APPROVED** [review]
only the 1 nit

File: app/src/components/pages/collectionsPage/collectionsPage.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply app/src/components/pages/collectionsPage/collectionsPage.tsx:164]: Hmm good point. I don't think you need the aria-label, the `labelText` should cover that.

**Review by EdwinGuzman — COMMENTED** [review]
Looks good but can you verify that the aria-label is not needed?  The labelText should add the value in the form element which is enough.

**Comment by avertrees** [comment]
@EdwinGuzman the aria-label was added for the a11y test and it did not fix the issue. I can remove it for now and the DS docs do say that the labelText should be enough to populate the aria label. I made another ticket for the a11y tests: [DR-3342 FE | Write a11y tests for all phase 2 pages](https://newyorkpubliclibrary.atlassian.net/browse/DR-3342?atlOrigin=eyJpIjoiMjA0ZTM3Nzk4MTc5NDE3YjkwYWQxYzYzY2ZjNDQ1ODQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**